.github/workflows/staging_e2e.yaml

```diff
@@ -0,0 +1,58 @@
+# Copyright 2024 NVIDIA CORPORATION
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: [Staging] End-to-end Tests
+
+on:
+  workflow_run:
+    workflows: [Image]
+    types:
+      - completed
+    branches:
+      - main
+      - release-*
+
+jobs:
+  staging-e2e:
+    runs-on: pdx01-arc-runners
+    if: ${{ github.event.workflow_run.conclusion == 'success' }}
```

Comment:
Question: How is the image made accessible to the tests in this context?

Reply:
umm we will have to push it. meaning we can't test during PR's unless we modify the image building. I totally forgot about that. Should we leave it for Merge events, as the currently running e2e?

Reply:
Yes, let's start there.

Reply:
done